samples/bluetooth/rpc_host/prj.conf

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BT_RPC=y
+CONFIG_BT_RPC_HOST=y
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_ECC=n
+# BT Device name must be the same like on the client side
+CONFIG_BT_DEVICE_NAME="Test beacon"
+
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_NRF_RPC_THREAD_STACK_SIZE=4096
+
+# Options for better debugging
+#CONFIG_LOG=y
+#CONFIG_NRF_RPC_LOG_LEVEL_DBG=y
+#CONFIG_NRF_RPC_TR_LOG_LEVEL_DBG=y
+#CONFIG_NRF_RPC_OS_LOG_LEVEL_DBG=y
+#CONFIG_BT_RPC_LOG_LEVEL_DBG=y
+
+#CONFIG_NO_OPTIMIZATIONS=y
+#CONFIG_DEBUG_OPTIMIZATIONS=y
+#CONFIG_DEBUG=y
+
+#CONFIG_HW_STACK_PROTECTION=y
```

Comment:
Please place this in a dedicated Kconfig fragment, `overlay-logging.conf`.

That way users can build this sample and enable logging without modifying the `prj.conf` by running:
```
west build -b nrf5340dk_nrf5340_cpunet -- -DCONFIG_OVERLAY=overlay-logging.conf
```